### ISSUE TITLE
How to use the pretrained models?

### ISSUE THREAD

**Issue by xieydd** [Tencent/PocketFlow#56]
I use the shell 
```
./scripts/run_loack.sh nets/resnet_at)ilsvrc12_run.py -n=4 --dcp_save_path=models/model.ckpt-250227.data-00000-of-00001
```
and the model.ckpt-250227.data-00000-of-00001 is from models_resnet50_at_ilsvrc_12.tar.gz from your office repository @jiaxiang-wu

**Comment by jiaxiang-wu** [comment]
Just download and unzip the pre-trained model files into the "./models" directory. No need to specify the `--dcp_save_path` argument.
Also, to use the pre-trained ResNet-50 model, you need to specify `--resnet_size 50`.

**Comment by xieydd** [comment]
Why i start train , the accuracy is so low `6.2500e-02`
